dist/static/patterns/pattern-1.svg

```diff
@@ -0,0 +1 @@
="pattern-1" class="map-pattern" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2709.66 2709.66"><path class="a" d="M604.83 304.83v-300l300 300h-300zM304.83 304.83h-300l300-300v300zM604.83 304.83a150 150 0 0 0-300 0zM604.83 604.83v-300l300 300h-300zM304.83 604.83h-300l300-300v300zM304.83 604.83a150 150 0 0 0 0-300zM604.83 604.83h300l-300 300v-300zM304.83 904.83h-300l300-300v300zM604.83 904.83a150 150 0 0 0-300 0zM1504.83 304.83v-300l300 300h-300z"/><path class="a" d="M1204.83 304.83h-300l300-300v300zM1504.83 304.83a150 150 0 0 0-300 0zM1504.83 604.83v-300l300 300h-300zM1204.83 604.83h-300l300-300v300zM1204.83 604.83a150 150 0 0 0 0-300zM1504.83 604.83h300l-300 300v-300zM1204.83 904.83h-300l300-300v300zM1504.83 904.83a150 150 0 1 0-300 0zM2404.83 304.83v-300l300 300h-300zM2104.83 304.83h-300l300-300v300zM2404.83 304.83a150 150 0 1 0-300 0zM2404.83 604.83v-300l300 300h-300zM2104.83 604.83h-300l300-300v300zM2104.83 604.83a150 150 0 0 0 0-300zM2404.83 604.83h300l-300 300v-300zM2104.83 904.83h-300l300-300v300zM2404.83 904.83a150 150 0 1 0-300 0zM604.83 1204.83v-300l300 300h-300zM304.83 1204.83h-300l300-300v300zM604.83 1204.83a150 150 0 0 0-300 0zM604.83 1504.83v-300l300 300h-300zM304.83 1504.83h-300l300-300v300zM304.83 1504.83a150 150 0 0 0 0-300zM604.83 1504.83h300l-300 300v-300zM304.83 1804.83h-300l300-300v300zM604.83 1804.83a150 150 0 0 0-300 0z"/><path class="a" d="M1504.83 1204.83v-300l300 300h-300zM1204.83 1204.83h-300l300-300v300zM1504.83 1204.83a150 150 0 0 0-300 0zM1504.83 1504.83v-300l300 300h-300zM1204.83 1504.83h-300l300-300v300zM1204.83 1504.83a150 150 0 0 0 0-300zM1504.83 1504.83h300l-300 300v-300zM1204.83 1804.83h-300l300-300v300zM1504.83 1804.83a150 150 0 0 0-300 0zM2404.83 1204.83v-300l300 300h-300zM2104.83 1204.83h-300l300-300v300zM2404.83 1204.83a150 150 0 0 0-300 0zM2404.83 1504.83v-300l300 300h-300zM2104.83 1504.83h-300l300-300v300zM2104.83 1504.83a150 150 0 0 0 0-300zM2404.83 1504.83h300l-300 300v-300zM2104.83 1804.83h-300l300-300v300zM2404.83 1804.83a150 150 0 0 0-300 0zM604.83 2104.83v-300l300 300h-300zM304.83 2104.83h-300l300-300v300zM604.83 2104.83a150 150 0 0 0-300 0zM604.83 2404.83v-300l300 300h-300zM304.83 2404.83h-300l300-300v300zM304.83 2404.83a150 150 0 0 0 0-300zM604.83 2404.83h300l-300 300v-300zM304.83 2704.83h-300l300-300v300zM604.83 2704.83a150 150 0 0 0-300 0zM1504.83 2104.83v-300l300 300h-300z"/><path class="a" d="M1204.83 2104.83h-300l300-300v300zM1504.83 2104.83a150 150 0 0 0-300 0zM1504.83 2404.83v-300l300 300h-300zM1204.83 2404.83h-300l300-300v300zM1204.83 2404.83a150 150 0 0 0 0-300zM1504.83 2404.83h300l-300 300v-300zM1204.83 2704.83h-300l300-300v300zM1504.83 2704.83a150 150 0 0 0-300 0zM2404.83 2104.83v-300l300 300h-300zM2104.83 2104.83h-300l300-300v300zM2404.83 2104.83a150 150 0 0 0-300 0zM2404.83 2404.83v-300l300 300h-300zM2104.83 2404.83h-300l300-300v300zM2104.83 2404.83a150 150 0 0 0 0-300zM2404.83 2404.83h300l-300 300v-300zM2104.83 2704.83h-300l300-300v300zM2404.83 2704.83a150 150 0 0 0-300 0z"/></svg>
```

Comment:
gulp-imagemin is just a stack for the 4 different npm modules including svgo https://www.npmjs.com/package/gulp-imagemin but I think we need to set the default config to not strip IDs at that is often the issue there. Lets leave for now but we should fix the starterpack so we can use src for svgs easily